numpy/_core/tests/test_shape_base.py

```diff
@@ -294,13 +294,17 @@ def test_exceptions(self):
         assert_raises(ValueError, concatenate, ())
 
     @pytest.mark.slow
-    @pytest.mark.skipif(sys.maxsize < 2**32, reason="only problematic on 64bit platforms")
+    @pytest.mark.skipif(
+        sys.maxsize < 2**32,
```

Comment:
32 bit platforms have `sys.maxsize == 2**31 - 1`, but I guess this also works :P